core/tests/tests.rs

```diff
@@ -200,6 +200,7 @@ fn added_subgraph_id_eq(
 }
 
 #[test]
+#[ignore]
```

Comment:
Why did this need to be ignored?

Reply:
Thank you for catching this! I did mean to do more here.

Reply:
Fixed